build/tools/Automation/checkPipelineStatus.sh

```diff
@@ -0,0 +1,40 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+# This script will check the pipeline status' result
+# for a given a pipeline invocation id.
+# A status.json file contains the meta data for a pipeline invocation id
+# of the point in time it was called, until the result has 'succeeded'.
+# The 'buildNumber' field of status.json is used later in the workflow
+# to update constants.yaml runtime image tag.
+
+pipelineInvocationId=$1
+maxRetries=3
+if [[ -n "${MAX_RETRIES}" ]]; then
+  maxRetries=${MAX_RETRIES}
+fi
+
+timeoutSeconds=10
```

Comment:
I'm just curious: here we're setting our defaults to take 40 seconds (10 seconds per try, 1 initial try + 3 retries). Is that realistic? Looking at the average time to run the pipeline, is it in this range? I would have expected it to be taking a lot more time than that, but it's good if it's closer to 30 seconds.

Reply:
I've updated the default time. Thanks for catching this!

Reply:
Thanks!